Ks sk683 update activation start10m

Kazarus4 opened the pull request:
Tech and peer review done
Translations are done
Links checked with gitpod and all in 200 ok
Ckecks with quality check page done 
I check the syntaxe (I hope it's ok for all markdown syntaxes)

Need proof reading for EN-GB and DE and IT-PT-ES too before merge.

Commit:
Update activationSTART10M need a proofreading

Commit:
Update fr-fr ActivationStart10M wait proofreading

Y0Coss committed:
Proofreading Start10M activation update

Commit:
tech review FR-FR done

Commit:
translation done

Commit:
links review gitpod and all translations done

Commit:
translations done need prfrd EN and DE

Commit:
translations and checks done need prfrd EN and DE

bs-ovh committed:
DE proof

bs-ovh committed:
EN proof

Commit:
Proofreading EN and DE update

OVH-Team-Guides commented on pages/web/hosting/activate_start10m/guide.es-es.md:
```diff
@@ -3,59 +3,86 @@ title: 'Activar el alojamiento web gratuito Start 10M'
 
+> [!warning]
+>
+> Si selecciona una de las dos casillas de verificación 'Entrada DNS A` y ' Entrada DNS MX` o ambas, la configuración inicial se vaciará en la zona DNS del dominio.
```
```suggestion
> Si selecciona una de las dos casillas de verificación `Entrada DNS A` y  `Entrada DNS MX` o ambas, la configuración inicial se vaciará en la zona DNS del dominio.
```

OVH-Team-Guides commented on pages/web/hosting/activate_start10m/guide.it-it.md:
```diff
@@ -3,28 +3,32 @@ title: "Attivare l'hosting gratuito Start 10M"
 > [!warning]
-> Questo hosting gratuito da 10 MB è utile per una semplice pagina Web di presentazione e **non include un database**. Per creare un sito Web composto da più pagine e che necessita di un database, come un CMS (ad esempio: WordPress, Joomla!, PrestaShop, Drupal...), ti consigliamo di ordinare una delle [nostre soluzioni di hosting direttamente dal nostro sito](https://www.ovhcloud.com/it/web-hosti ...
+>
+> Questo hosting gratuito da 10 MB è ideale per una semplice pagina Web di presentazione. **non include database**.
```
```suggestion
> Questo hosting gratuito da 10 MB è ideale per una semplice pagina Web di presentazione, **non include database**.
```

OVH-Team-Guides commented on pages/web/hosting/activate_start10m/guide.pl-pl.md:
```diff
@@ -38,24 +43,47 @@ Pojawi się okno aktywacyjne. W **etapie 1** wyświetla się przypomnienie ofert
 
+> [!warning]
+>
+> Jeśli zaznaczysz jedno z dwóch pól `End DNS` i `End DNS` lub oba te pola, zostanie on usunięty z konfiguracji pierwotnie wprowadzonej w strefie DNS Twojej domeny.
```
something's wrong here, it should mention `DNS A record` and `DNS MX record`

Kazarus4 committed:
Update pages/web/hosting/activate_start10m/guide.es-es.md

Co-authored-by: OVH-Team-Guides <43037584+OVH-Team-Guides@users.noreply.github.com>

Kazarus4 committed:
Update pages/web/hosting/activate_start10m/guide.it-it.md

Co-authored-by: OVH-Team-Guides <43037584+OVH-Team-Guides@users.noreply.github.com>

Kazarus4 committed:
`Rekord DNS A` i `Rekord DNS MX`

`Rekord DNS A` i `Rekord DNS MX`

Kazarus4 committed:
requirments

- A [domain name](https://www.ovhcloud.com/en-gb/domains/){.external} in your OVHcloud Control Panel, detached from any web hosting plan, and with no associated [MX Plan](https://docs.ovh.com/gb/en/emails/web_hosting_an_overview_of_ovh_email/).
- Access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB){.external}

Kazarus4 committed:
requirment

- A [domain name](https://www.ovhcloud.com/en-ie/domains/){.external} in your OVHcloud Control Panel, detached from any web hosting plan, and with no associated [MX Plan](https://docs.ovh.com/ie/en/emails/web_hosting_an_overview_of_ovh_email/).
- Access to the [domain name](https://www.ovhcloud.com/en-ie/domains/){.external} in your [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.ie/&ovhSubsidiary=ie){.external}
